src/theories/Category.jl

```diff
@@ -102,45 +67,45 @@ end
 
 Axiomatized as a covariant category action.
 """
-@theory ThCopresheaf{Ob,Hom,El} <: ThCategory{Ob,Hom} begin
+@theory ThCopresheaf <: ThCategory begin
   # copresheaf = object-indexed family
   El(ob::Ob)::TYPE
 
   # functoriality = covariant action
-  act(x::El(A), f::Hom(A,B))::El(B) ⊣ (A::Ob, B::Ob)
-  @op (⋅) := act
+  act(x::El(A), f::Hom(A,B))::El(B) ⊣ [A::Ob, B::Ob]
+  # @op (⋅) := act
```

Comment:
Is this still unsupported? If so, that's fine but let's make sure to have an issue tracking it if we don't already.

Reply:
I'm not sure if we *want* to support this. The way aliases work right now is that they are handled at the scope level, not at the theory level. We'd have to move aliases to the theory level in order to support this, because when something is an alias for two different functions, we'd have to dispatch to resolve which one should be used for a particular invocation of the alias. This isn't used very much in Catlab, so might not be worth complicating the design over.

Reply:
That seems like a reasonable position to take. In that case, though, if we're not planning to support it, then the theories in Catlab currently using aliases in this way should be refactored, i.e., choosing different aliases or rewriting the axioms to not use the aliases.

Another idea, which might be too crazy, is to reduce all aliases to term constructors supplemented with equality axioms. Arguably that is more uniform but I could imagine it causing computational annoyances, e.g., alias equalities junking up e-graphs.